integrations/access/opsgenie/client_test.go

```diff
@@ -68,8 +69,11 @@ func TestCreateAlert(t *testing.T) {
 		Message:     "Access request from someUser",
 		Alias:       "teleport-access-request/someRequestID",
 		Description: "someUser requested permissions for roles role1, role2 on Teleport at 01 Jan 01 00:00 UTC.\nReason: someReason\n\n",
-		Responders:  []Responder{{Type: "schedule", ID: "responder@teleport.com"}},
-		Priority:    "somePriority",
+		Responders: []Responder{
+			{Type: "schedule", ID: "responder@teleport.com"},
```

Comment:
While you're here, can you update references to `teleport.com` to `goteleport.com` or `example.com`? We shouldn't use domains we don't own anywhere in our code, tests or otherwise.